### DISCUSSION TITLE
Wool to string

### DISCUSSION THREAD

**Discussion by NotEvenAUnicorn** [PurpurMC/Purpur#649] [__Dismissed__] [answered]
I know there are data packs/plugins/mods that do this, but it'd be nice to just not have to add anything else to the server.

String --> Wool
Wool --> String

**Comment by granny** [comment]
I'm not sure what it is you're requesting. A recipe that allows you to turn string into wool and vice versa? If that's the case, then no, it's not happening since recipes are better handled by using a datapack.

**Reply by NotEvenAUnicorn** [reply]
Was worth a shot lol. Thanks!

**Comment by craftish37** [comment, marked as answer]
Here's something that can help you if you don't want to mind about the json syntax 👀
https://crafting.thedestruc7i0n.ca
